src/main/drivers/pwm_output_dshot.c

```diff
@@ -126,7 +126,11 @@ static void motor_DMA_IRQHandler(dmaChannelDescriptor_t *descriptor)
 void pwmDshotMotorHardwareConfig(const timerHardware_t *timerHardware, uint8_t motorIndex, motorPwmProtocolTypes_e pwmProtocolType, uint8_t output)
 {
 #if defined(STM32F3)
+#ifdef USE_DSHOT_DMAR
```

Comment:
I am wondering if this was easier to read if `#ifdef USE_DSHOT_DMAR` was the outer condition?